Infra/Repositorios/RepositorioPersonagem.cs

```diff
@@ -0,0 +1,38 @@
+using Dominio.Interfaces;
+using Dominio.Modelos;
+using Testes.Interfaces;
+
+namespace Infra.Repositorios;
+
+public class RepositorioPersonagem : IRepositorioPersonagem
+{
+    public List<Personagem> ObterTodos()
+    {
+        throw new NotImplementedException();
+    }
+
+    public Personagem ObterPorId(int id)
+    {
+        throw new NotImplementedException();
+    }
+
+    public void Criar(Personagem t)
+    {
+        throw new NotImplementedException();
+    }
+
+    public Personagem Editar(Personagem t)
+    {
+        throw new NotImplementedException();
+    }
+
+    public void Deletar(int id)
+    {
+        throw new NotImplementedException();
+    }
+
+    public IEnumerable<Personagem> ObterPersonagensPorRaca(Raca raca)
```

Comment:
esse é o seu metodo para filtragem? se for, nao seria exatamente assim o esperado e sim que vc coloque um parametro de filtro no obter todos

Reply:
Tinha me confundido com o exemplo do link no treinamento. Corrigido!